package.json

```diff
@@ -13,53 +13,70 @@
-    "rimraf": "^2.6.1",
-    "rollup": "^0.41.6",
-    "rollup-plugin-babel": "^2.7.1",
-    "rollup-plugin-json": "^2.1.1",
-    "rollup-watch": "^3.2.2",
-    "videojs-standard": "^5.2.0"
+    "conventional-changelog-cli": "^2.0.1",
+    "conventional-changelog-videojs": "^3.0.0",
+    "doctoc": "^1.3.1",
+    "husky": "^1.0.0-rc.13",
+    "in-publish": "^2.0.0",
+    "jsdoc": "https://github.com/BrandonOCasey/jsdoc#feat/plugin-from-cli",
+    "karma": "^3.0.0",
+    "lint-staged": "^7.2.2",
+    "nodeunit": "^0.11.3",
+    "not-prerelease": "^1.0.1",
+    "npm-merge-driver-install": "^1.0.0",
+    "npm-run-all": "^4.1.3",
+    "pkg-ok": "^2.2.0",
+    "rollup": "^0.65.0",
+    "shx": "^0.3.2",
+    "sinon": "^6.1.5",
+    "videojs-generate-karma-config": "~3.0.0",
+    "videojs-generate-rollup-config": "~2.2.0",
+    "videojs-standard": "^7.1.0"
   },
   "scripts": {
     "prebuild": "npm run clean",
     "build": "npm-run-all -p build:*",
-    "build:js": "npm-run-all -p build:js:modules build:js:umd build:js:pad build:js:unpad",
-    "build:js:modules": "rollup -c scripts/modules.rollup.js",
-    "build:js:umd": "rollup -c scripts/umd.rollup.js",
-    "build:js:pad": "rollup -c scripts/pad.rollup.js",
-    "build:js:unpad": "rollup -c scripts/unpad.rollup.js",
-    "build:test": "rollup -c scripts/test.rollup.js",
-    "clean": "rimraf dist test/dist",
-    "postclean": "mkdirp dist test/dist",
+    "build:js": "rollup -c scripts/rollup.config.js",
+    "clean": "shx rm -rf ./dist ./test/dist",
+    "postclean": "shx mkdir -p ./dist ./test/dist",
+    "docs": "npm-run-all docs:*",
+    "docs:api": "jsdoc src -g plugins/markdown -r -d docs/api",
+    "docs:toc": "doctoc README.md",
     "lint": "vjsstandard",
-    "pretest": "npm-run-all lint build:test",
-    "test": "nodeunit test/dist/pkcs7_test.js",
+    "server": "karma start scripts/karma.conf.js --singleRun=false --auto-watch",
+    "start": "npm-run-all -p server watch",
+    "pretest": "npm-run-all lint build",
+    "test": "npm-run-all -p test:browser test:node",
+    "test:browser": "karma start scripts/karma.conf.js",
+    "test:node": "qunit test/dist/bundle.js",
+    "posttest": "shx cat test/dist/coverage/text.txt",
+    "update-changelog": "conventional-changelog -p videojs -i CHANGELOG.md -s",
+    "preversion": "npm test",
+    "version": "is-prerelease || npm run update-changelog && git add CHANGELOG.md",
     "watch": "npm-run-all -p watch:*",
-    "watch:js": "npm-run-all -p watch:js:modules watch:js:umd watch:js:pad watch:js:unpad",
-    "watch:js:modules": "rollup -c scripts/modules.rollup.js -w",
-    "watch:js:umd": "rollup -c scripts/umd.rollup.js -w",
-    "watch:js:pad": "rollup -c scripts/pad.rollup.js -w",
-    "watch:js:unpad": "rollup -c scripts/unpad.rollup.js -w",
-    "watch:test": "rollup -c scripts/test.rollup.js -w",
-    "prepublish": "npm run build"
+    "watch:js": "npm run build:js -- -w",
+    "watch:test": "npm run test:node -- -w",
```

Comment:
This watches the tests and re-runs them when `test/dist/bundle.js` changes, I could see removing this or keeping it in. It mostly to have parity when `npm run start` is used so that we get test feedback on changes.